providers/aws/services/iam/iam_user_hardware_mfa_enabled/iam_user_hardware_mfa_enabled.py

```diff
@@ -0,0 +1,41 @@
+from lib.check.models import Check, Check_Report
+from providers.aws.services.iam.iam_service import iam_client
+
+
+class iam_user_hardware_mfa_enabled(Check):
+    def execute(self) -> Check_Report:
+        findings = []
+        response = iam_client.users
+
+        if response:
+            for user in response:
+                report = Check_Report(self.metadata)
+                report.resource_id = user.name
+                report.resource_arn = user.arn
+                report.region = "us-east-1"
+                if user.mfa_devices:
+                    for mfa_device in user.mfa_devices:
+                        if mfa_device.type == "mfa" or mfa_device.type == "sms-mfa":
+                            report.status = "FAIL"
+                            report.status_extended = f"User {user.name} has a virtual MFA instead of a hardware MFA enabled."
```

Comment:
```suggestion
                            report.status_extended = f"User {user.name} has not a hardware MFA enabled."
```

Reply:
I think is better to give this kind of information to the user.